src/styles.scss

```diff
@@ -23,3 +23,11 @@ body {
   box-shadow: none;
   border: $input-border-width solid $input-border-color;
 }
+
```

Comment:
ouch! Could we find the other z-index that is below the time picker? so we won't use this large z-index?

Reply:
I have changed to 1100 the minimum value